srcpkgs/rpi-kernel/template

```diff
@@ -12,12 +12,12 @@
 #
 # WARNING: keep all rpi*-kernel packages in sync
```

Comment:
also update `rpi{2,3,4}-kernel`

Reply:
I don't have those devices, so I refrained from doing that. Perhaps divergence is not a bad idea, considering that the previous update broke, these often headless and remote, systems quite severely.

Reply:
no, they need to be kept in sync